src/sample/View.java

```diff
@@ -99,51 +101,47 @@ public void changed(ObservableValue<? extends Boolean> observable, Boolean oldVa
 
     private void prepTheGame(Text helpText, CheckBox defaults, TextField textInput, Stage startGUI)
     {
+        startGUI.close();
         int gridSize;
-
+            // if defaults are checked then apply the default 5x5 size
             if (defaults.isSelected())
             {
                 textInput.clear();
                 gridSize = 5;
```

Comment:
There's a constants file for this, I think we should reference that instead of hardcoding values deep down.  We should do that for all numeric values

Reply:
Fair enough. I can fix that real quick.

Reply:
while I was at it, I moved the max size message into a pop up alert GUI.